Python/tier2.c

```diff
@@ -113,12 +113,17 @@ _PyTier2TypeContext_Copy(const _PyTier2TypeContext *type_context)
             _Py_TYPENODE_t *parent = (_Py_TYPENODE_t *)_Py_TYPENODE_CLEAR_TAG(node);
 
             // Check if part of locals
-            if (parent - type_context->type_locals < nlocals) {
-                type_locals[i] = node - (uintptr_t)orig_type_locals + (uintptr_t)type_locals;
+            int plocals = (int)(parent - type_context->type_locals);
```

Comment:
Could we use another name please? `plocals` signals its a pointer but it's not, it's just offset. So `offset_locals` maybe?